lib/ui/nav_pages/home_page/home_page_viewmodel.dart

```diff
@@ -111,6 +111,10 @@ class HomePageViewModel extends StreamViewModel {
     NavigationService().navigateTo(Routes.fileSearchView);
   }
 
+  void navigateToChannelPage() {
+    NavigationService().navigateTo(Routes.channelPageView);
```

Comment:
make the NavigationService a field in the class

Reply:
done